### DISCUSSION TITLE
Suggest a missing mod or a map.

### DISCUSSION THREAD

**Discussion by malortie** [HL2-Mods-Legacy-Project/hl2-legacy-main#91] [General]
If you don't see a mod in the [list](https://github.com/HL2-Mods-Legacy-Project/hl2-legacy-main/issues/90), comment here and I'll add it. You can also open an issue on the repository.

### Rules

This must be a **Pre-Steampipe** mod and one of the following:

- Singleplayer HL2/EP1/EP2/Lost coast mod
- Standalone Singleplayer HL2/EP1/EP2/Lost coast maps
- Multiplayer HL2/EP1/EP2/Lost coast mod

State if it's a mod or a map.

### Multiplayer mods

Singleplayer mods are foremost prioritized. Multiplayer mods might be added for preservation but it will likely be limited to only making them launchable and won't have further fixes since it requires extra maintenance, testers, setting up servers, etc.

### Portal 1 mods

Pre-Steampipe Portal 1 mods can be suggested but are not guaranteed to be added. Portal 1's source code was never officially released so there's no way to legally make Source 2007 compatible binaries for mods.

**Comment by dougjp11** [comment]
Hi. Thank you for your excellent work! For example just now I was able to play Eye of the Storm for HL2 Ep2 again. 

And thanks for taking suggestions for maps/mods that won't play because of the 2013 mess. These won't play:

Half-Life 2 Episode 2:
– Cyberscape
– Depot
– Hard Escape
– Intrusion
– Mesa Fallback
– Mission to Mars
– Opposing Force: Ravenholm
– Sity
– Unexpected Conclusion
– Uzvara
– Zombie Escape

Half-Life 2:
- Escape ExMo
- School Adventures
- ASDA
- City Rebellion
- Deal With Destiny
- Outpost Xen
- Rebellion
- Half Life 2 Sewers
- The Island
- The Lighthouse
- The Sewer

**Comment by malortie** [comment]
@dougjp11 

Thanks, I'll review each and add them when applicable.

Note: The following were already fixed:

- School Adventures
- Rebellion
- The Island

See #1.

**Reply by dougjp11** [reply]
Fantastic! Thanks again so much for your work on this. I don't have the skills, and for example every time I see somebody quote revised gameinfo.txt in a forum, I can never make it work.

**Reply by malortie** [reply]
@dougjp11

City Rebellion is done: https://github.com/HL2-Mods-Legacy-Project/hl2-legacy-mod-city-rebellion/releases/tag/v1.0.0

Make sure you installed [Source SDK Base Resources](https://github.com/Source-SDK-Base-Legacy-Project/Source-SDK-Base-Resources/blob/master/docs/INSTALL.md) (patches Source SDK Base(s) + installs vanilla HL2 binaries).

**Reply by malortie** [reply]
Intrusion is done: https://github.com/HL2-Mods-Legacy-Project/hl2-legacy-mod-intrusion/releases/tag/v1.0.0

**Reply by malortie** [reply]
Outpost Xen is [done]( https://github.com/HL2-Mods-Legacy-Project/hl2-legacy-mod-outpost-xen/releases/tag/v1.0.0). See this [issue]( https://github.com/HL2-Mods-Legacy-Project/hl2-legacy-mod-outpost-xen/issues/1) too.

**Reply by malortie** [reply]
Mesa Fallback is done: https://github.com/HL2-Mods-Legacy-Project/hl2-legacy-mod-mesa-fallback/releases/tag/v1.0.0

**Reply by malortie** [reply]
Deal With Destiny is done: https://github.com/HL2-Mods-Legacy-Project/hl2-legacy-mod-deal-with-destiny/releases/tag/v1.0.0

**Reply by malortie** [reply]
The Lighthouse is done: https://github.com/HL2-Mods-Legacy-Project/hl2-legacy-mod-the-lighthouse/releases/tag/v1.0.0

**Comment by andrejkuroglo8** [comment]
Hi, You can add this mod:
https://www.moddb.com/mods/platform ?

**Reply by malortie** [reply]
@andrejkuroglo8

Added to [v0.5.0](https://github.com/malortie/SourceModPatcher/milestone/1)'s list.

I'll look into it after Source SDK Base Resources (New SourceModPatcher) is released. See [this thread](https://github.com/malortie/SourceModPatcher/issues/102) for more info.

**Reply by malortie** [reply]
@andrejkuroglo8

Platform is done: https://github.com/HL2-Mods-Legacy-Project/hl2-legacy-mod-platform/releases/tag/v1.0.0

Make sure you installed [Source SDK Base Resources](https://github.com/Source-SDK-Base-Legacy-Project/Source-SDK-Base-Resources/blob/master/docs/INSTALL.md) (patches Source SDK Base(s) + installs vanilla HL2 binaries).
